app/models/device.rb

```diff
@@ -0,0 +1,80 @@
+class Device < ApplicationRecord
+  belongs_to :user
+
+  IOS = "iOS".freeze
+  ANDROID = "Android".freeze
+
+  validates :token, uniqueness: { scope: %i[user_id platform app_bundle] }
+  validates :platform, inclusion: { in: [IOS, ANDROID] }
+
+  def create_notification(title, body, payload)
+    case platform
+    when IOS
+      ios_notification(title, body, payload)
+    when ANDROID
+      android_notification(title, body, payload)
+    end
+  end
+
+  private
+
+  def ios_notification(title, body, payload)
+    n = Rpush::Apns2::Notification.new
+    # [@forem/backend] `.where().first` is necessary because we use Redis data storage
+    # https://github.com/rpush/rpush/wiki/Using-Redis#find_by_name-cannot-be-used-in-rpush-redis
+    # rubocop:disable Rails/FindBy
+    n.app = Rpush::Apns2::App.where(name: app_bundle).first || recreate_ios_app!
+    # rubocop:enable Rails/FindBy
+
+    n.device_token = token
+    n.data = {
+      aps: {
+        alert: {
+          title: ApplicationConfig["COMMUNITY_NAME"],
+          subtitle: title,
+          body: body
+        },
+        'thread-id': ApplicationConfig["COMMUNITY_NAME"]
```

Comment:
odd that Rubocop hasn't complained about the single quote

Reply:
I think Rubocop knows in this case you HAVE to use single quotes for a symbol with a dash